sdk/include/event.h

```diff
@@ -98,3 +121,24 @@ int __cheri_compartment("sched")
   event_delete(struct SObjStruct *heapCapability, void *evt);
 
 __END_DECLS
+
+#ifdef __cplusplus
```

Comment:
Not sure these should go in the public header: I don't see them as being useful for anything other than the scheduler (a helper for constructing the flags would be).